merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -178,15 +188,15 @@ public List<String> validateModelArguments(final String missionModelId, final Ma
          InvalidArgumentsException
   {
     return this.loadUnconfiguredMissionModel(missionModelId)
-               .validateMissionModelArguments(arguments);
+        .getConfigurationType()
+        .validateArguments(arguments);
```

Comment:
Very minor nitpick, but the indentation here is just a little inconsistent. I'd expect the method call dots to be vertically aligned, either in the way it was before (with the first method on the same line as `this`) or by moving the first method onto its own line.

This happens in a few places in this commit ("Inline MissionModelFacade.Unconfigured out of existence"). Don't worry about it unless it really bothers you too.